terraform/utils.go

```diff
@@ -87,6 +87,17 @@ func tfConfigList(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateDat
 
 	// Sanitize the matches to likely Terraform files
 	for _, i := range matches {
+		// Check file or directory
+		fileInfo, err := os.Stat(i)
+		if err != nil {
+			plugin.Logger(ctx).Error("fileList", "error reading file path", err)
```

Comment:
`plugin.Logger(ctx).Error("tfConfigList", "failed to get file info", err)`

We should in general use the function name as the first arg, then follow it up with error message and then the error